BeatSaberPlaylistsLib/Types/IPlaylist.cs

```diff
@@ -30,10 +30,6 @@ public partial interface IPlaylist : IList<IPlaylistSong>
         /// </summary>
         string? SuggestedExtension { get; set; }
 
-        /// <summary>
-        /// Custom data not included in the schema
-        /// </summary>
-        public Dictionary<string, object>? CustomData { get; set; }
```

Comment:
IPlaylist should have a way to get custom data (I agree the Dictionary in the interface needs to go). Perhaps `public bool TryGetCustomData(string key, out object value)` or `public bool TryGetCustomData<T>(string key, out T value)`. Playlist types without custom data would just always return false.